editor/src/components/editor/actions/actions.tsx

```diff
@@ -2229,10 +2231,14 @@ export const UPDATE_FNS = {
           success.imports,
           action.importsToAdd,
         )
+
+        // TODO handle duplicate name mapping
```

Comment:
Yes, these are places that use `mergeImports` but does not change the element name (for now). Since this PR is pretty large (changing the entire imports mechanism and fixing reparenting - which is a large and very delicate flow), I wanted to merge it, and then have subsequent very small PRs to handle other cases in the code (copy/paste, wrap, etc).